app.rb

```diff
@@ -1,20 +1,226 @@
 require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
+
 apartments = data[:apartments]
 tenants = data[:tenants]
+ruby_apartments = []
+ruby_tenants = []
+
+puts apartments
+# loop over apartments
 
-# First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
-  # Explain how the data is structured
-  # What are the properties for each of the two types of hashes
+apartments.each do |apartment|
+  Apartment.new
```

Comment:
when we run the new method, it needs the same arguments that you defined in the initialize method of the class definition.

In this case it needs the arguments :  `:id, :address, :rent, :square_feet`